scipy/optimize/_differentialevolution.py

```diff
@@ -588,8 +774,8 @@ def _mutate(self, candidate):
 
         fill_point = self.random_number_generator.randint(0, parameter_count)
 
-        if (self.strategy == 'randtobest1exp'
-                or self.strategy == 'randtobest1bin'):
```

Comment:
When I originally wrote this code I had the `or` on the first line.  Stylistic feedback that was given to me was that it's better to have it on the continuation line.  Can you roll back the changes on (new)lines:
777
840
481